### PR TITLE
remove ROLE_WEBAPPS_PERMISSIONS and related code

### PR DESCRIPTION
**Pull request by snopoke**
## Summary
Remove ICDS feature flag for restricting web apps by user role.

Opening for review pending check in with the one project that may be using this on prod.

## Feature Flag
ROLE_WEBAPPS_PERMISSIONS (role_webapps_permissions): ICDS: Toggle which webapps to see based on role

## Product Description
Remove the ability to restrict which apps are accessible to users in Web Apps based on the user's role.

@dimagi/product the alternative to to GA this if you want this ability in the product.

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### QA plan
Tested locally:

- create role
- update role (with and without the permission)
- delete role (with and without the permission)
- use web apps (with and without the permission in the role)

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

**Comment by orangejenny** [comment]
@AmitPhulera Could you review this? I think you're the person who has most recently worked with this feature.

File: corehq/apps/cloudcare/views.py [hunk 1]
```diff
@@ -131,9 +131,6 @@ def get_web_apps_available_to_user(self, domain, user):
         except DomainMembershipError:
```
Comment (AmitPhulera) [inline corehq/apps/cloudcare/views.py:131]: Should we completely remove `role` from this function? 
I am not sure if it's serving any purpose here?

**Review by AmitPhulera — APPROVED** [review]

File: corehq/apps/cloudcare/views.py (hunk 1 above)
Comment (snopoke) [reply corehq/apps/cloudcare/views.py]: 1d8a8737e8f

**Comment by snopoke** [comment]
Got approval from @dimagi/product to remove.

**Comment by snopoke** [comment]
oops, build was still going - I'll monitor build and revert this if it fails

**Comment by snopoke** [comment]
That build failed due to branch being removed. Triggered new build on master: https://travis-ci.com/github/dimagi/commcare-hq/builds/226700839

**Pull request merged**